samples/sensor/2_apds9960/testcase.ini

```diff
@@ -0,0 +1,5 @@
+[test]
+build_only = true
+tags = samples
+arch_whitelist = arc
```

Comment:
remove the arch_whitelist, it is not needed, this needs to work on all architectures.